mindmeld/constants.py

```diff
@@ -15,3 +15,43 @@
 BLUEPRINTS_URL = "https://blueprints.mindmeld.com"
 BINARIES_URL = "https://binaries.mindmeld.com"
 DUCKLING_VERSION = "20200701"
+
+SPACY_ANNOTATOR_SUPPORTED_ENTITIES = [
+    "sys_time",
+    "sys_interval",
+    "sys_duration",
+    "sys_number",
+    "sys_amount-of-money",
+    "sys_distance",
+    "sys_weight",
+    "sys_ordinal",
+    "sys_quantity",
+    "sys_percent",
+    "sys_org",
+    "sys_loc",
+    "sys_person",
+    "sys_gpe",
+    "sys_norp",
+    "sys_fac",
+    "sys_product",
+    "sys_event",
+    "sys_law",
+    "sys_langauge",
+    "sys_work-of-art",
+    "sys_other-quantity",
+]
+
+
+def _no_overlap(entity_one, entity_two):
```

Comment:
should this be a function of the entity itself `entity_one.has_overlap(entity_two)`?

Reply:
If there are future applications that would encounter overlapping entities, it makes sense to add this as a function of the class. (If so, which entity Class would this function be added to?) Otherwise, it may be okay to keep it as a separate method